client-lib/go/client/jvs_client_test.go

```diff
@@ -76,65 +76,56 @@ func TestValidateJWT(t *testing.T) {
 	if err != nil {
 		t.Fatal("couldn't create jwks json")
 	}
+	tok := testCreateToken(t, "test_id")
```

Comment:
I think you can inline these variable initialization to have cleaner code. Does the example below work?

Reply:
I inline the variable initialization for variables that will be used only once.